src/SRD.H

```diff
@@ -14,7 +14,8 @@
 
 
 //=============== DATA TYPES: dttab.cpp
-
+#ifndef SRD_H
```

Comment:
Seems strange that the comment above is outside the guard when it clearly is related to the comments below it.